opentracing/src/main/java/opentracing/Span.java

```diff
@@ -25,16 +25,7 @@
   /**
    * Suitable for serializing over the wire, etc.
    */
-  TraceContext traceContext();
-
-  /**
-   * Denotes the beginning of a subordinate unit of work.
-   *
-   * @param operationName name of the operation represened by the new span from the perspective of
-   * the current service.
-   * @return a new child Span in "started" state.
-   */
-  Span startChild(String operationName);
+  SpanContext childContext();
```

Comment:
> > -  Span startChild(String operationName);
> > +  SpanContext childContext();
>
> Some libraries divorce the concern of creating a span vs adding to it.
>
> By removing Span startChild(String operationName); are we not helping to solve this.
> Keeping the concern of creating spans in the Tracer.

I suppose I inferred that SpanContext childContext() was still an
operation to create a child span? Did you mean SpanContext context()?